### DISCUSSION TITLE
Recognition question

### DISCUSSION THREAD

**Discussion by A380Coding** [jomjol/AI-on-the-edge-device#2572] [General]
On my meter the last digit that was recognizes as value is a 3. xxx.yy3
On the image it is clearly a 4. Of course this is dependent on the alignment on the image, model, recognition and so forth.

What is not quite clear to me is the following:
I check under Data->Recognition. As the last digit of "recognized digit ROIs (last round) I can see the image "4" and above that it shows "4.0"
Doesn't that say the model recognized a 4? Why is the result 3 in the final value?
![ai](https://github.com/jomjol/AI-on-the-edge-device/assets/80172816/b65fd661-3de4-4b5a-95e2-342ecf9fb8cd)

**Comment by caco3** [comment]
To my understanding, in the recognition page each digit is shown with its individual recognition.
Howe ever in the combination there are some aditional checks.

Eg. Dig2 might show `3.5`, whereas Dig3 in fact is a `4.x`. This then leads to a value of `34x`.

**Comment by Slider0007** [comment]
This is most likely due to rounding in recognition page vs truncating by algo -> see #2540
